docs/workflow/requirements/linux-requirements.md

```diff
@@ -20,9 +20,8 @@ Below are the requirements for toolchain setup, depending on your environment. P
 
 Minimum RAM required to build is 1GB. The build is known to fail on 512 MB VMs ([dotnet/runtime#4069](https://github.com/dotnet/runtime/issues/4069)).
```

Comment:
Could you please also update the index at the top of the doc to match?

Reply:
Yikes, sorry for the lack of attention to detail. Thanks for all the suggestions.